### PR TITLE
SCP-1471: Marlowe Playground: dont show save, save as and rename in the home page

### PR DESCRIPTION
**Pull request by hrajchert**
This PR solves issue [SCP-1471](https://jira.iohk.io/projects/SCP/issues/SCP-1471).

Screen Shot 2020-11-16 at 17 00 40

Screen Shot 2020-11-16 at 17 00 49

Pre-submit checklist:
- Branch
    - [X] Commit sequence broadly makes sense
    - [X] Key commits have useful messages
    - [X] Relevant tickets are mentioned in commit messages
- PR
    - [X] Self-reviewed the diff
    - [X] Useful pull request description
    - [X] Reviewer requested
- If you changed any Purescript files:
   - [X] `$(nix-shell shell.nix --run fix-purty)` to fix any formatting issues

Pre-merge checklist:
- [x] Someone approved it
- [x] Commits have useful messages
- [x] Review clarifications made it into the code
- [x] History is moderately tidy; or going to squash-merge

### CODE REVIEW

File: marlowe-playground-client/src/MainFrame/View.purs [hunk 1]
```diff
@@ -187,6 +189,17 @@ menuBar state =
     else
       menuButton (OpenModal GithubLogin) name
 
+  -- Even if we end up writting more by selecting the cases in which the buttons should
+  -- appear, I prefer this to selecting which views we shouldn't show the buttons. The
+  -- reason is that if we add a new view, we don't need to adjust this.
+  -- TODO: Check if we should show the buttons for WalletEmulator and ActusBlocklyEditor
```
Comment (palas) [inline marlowe-playground-client/src/MainFrame/View.purs:195]: Good point, it seems the ACTUS Labs currently only saves Haskell, Javascript, and Marlowe. So better not to show it for now.
I cannot even find a way to access the Wallet, so it probably shouldn't have them either.

**Review by palas — APPROVED** [review]
LGTM

File: marlowe-playground-client/src/MainFrame/View.purs (hunk 1 above)
Comment (shmish111) [reply marlowe-playground-client/src/MainFrame/View.purs:195]: you can access wallets with the URL `/#/wallets`

Comment (shmish111) [reply marlowe-playground-client/src/MainFrame/View.purs:195]: but I agree, shouldn't be visible on those

**Review by shmish111 — APPROVED** [review]
LGTM

File: marlowe-playground-client/src/MainFrame/View.purs (hunk 1 above)
Comment (hrajchert) [reply marlowe-playground-client/src/MainFrame/View.purs:195]: how about ActusBlocklyEditor?

Comment (hrajchert) [reply marlowe-playground-client/src/MainFrame/View.purs:195]: @shmish111  while doing the home page redesign, I found that on line 121-138 there is a `otherActions` helper that also appends items to the topBar depending on the view. Do you think it makes sense in a following PR to move that inside `menuBar`?

**Pull request merged**